spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -105,6 +105,44 @@ def add_group_to_build(app_id: nil, group_id: nil, build_id: nil)
       handle_response(response)
     end
 
+    # Returns a list of available testing groups
+    # e.g.
+    #   {"b6f65dbd-c845-4d91-bc39-0b661d608970" => "Boarding",
+    #    "70402368-9deb-409f-9a26-bb3f215dfee3" => "Automatic"}
+    def groups(app_id)
+      return @cached_groups if @cached_groups
+
+      r = request(:get, "/testflight/v2/providers/#{self.provider.provider_id}/apps/#{app_id}/groups")
+      @cached_groups = parse_response(r, 'data')
+    end
+
+    #####################################################
+    # @!group Testers
+    #####################################################
+    def testers(tester)
```

Comment:
Not sure I understand what this is doing? Passing a tester to get the URL?

Reply:
yeah, it's copypasta from `Tunes` into `TestFlight` since `Tunes` should be deprecated/removed.